src/cmds/docs/index.js

```diff
@@ -48,7 +48,10 @@ exports.run = async function (opts) {
     return Promise.reject(new Error(`No folder provided. Usage \`${config.cli} ${exports.usage}\`.`));
   }
 
-  const selectedVersion = await getProjectVersion(version, key, true).catch(e => {
+  // TODO: should we allow version selection at all here?
+  // Let's revisit this once we re-evaluate our category logic in the API.
+  // Ideally we should ignore this parameter entirely if the category is included.
+  const selectedVersion = await getProjectVersion(version, key, false).catch(e => {
     return Promise.reject(e);
```

Comment:
Do we need to handle this rejection here? I think this might cause the same exit code issue you fixed last month.

Reply:
Oh yep you're right, thanks!

Reply:
fixed in https://github.com/readmeio/rdme/pull/421/commits/fe257a83c15816a5178aacebf1d5207c02975b3a